templates/fluent-resource/en-US/cli.ftl

```diff
@@ -0,0 +1,101 @@
+### Translation file for page: https://www.rust-lang.org/what/cli
+
+
+## templates/what/cli.hbs
+
+cli-page-heading = Command-line apps
+
+
+## templates/components/what/cli/pitch.hbs
+
+cli-pitch-heading = Why Rust?
+
+cli-pitch-solid-img-alt = Shield with a checkmark
+cli-pitch-solid-heading = Solid and quick
+cli-pitch-solid-description = Even if you’re just writing a short one-off
+        script, you can be confident it’s fast, easily testable, and gives
+        helpful output.
+cli-pitch-solid-link = Rust’s guarantees
+
+cli-pitch-ship-img-alt = box with a checkmark
+cli-pitch-ship-heading = Easy distribution
+cli-pitch-ship-description = Compile everything down to a single binary&mdash;no
+        need for your users to have a runtime or libraries installed.
+cli-pitch-ship-link = How to ship Rust code
+
+cli-pitch-config-img-alt = A note and a gear
+cli-pitch-config-heading = Robust configuration
+cli-pitch-config-description = Handle configuration files across platforms with
+        ease.  Rust will deal with namespaces and formats for you.
```

Comment:
These 2 spaces before "Rust will deal..." were in the original markup.

Reply:
It's HTML, it will be collapsed